src/scaffoldmaker/meshtypes/meshtype_3d_brainstem.py

```diff
@@ -0,0 +1,355 @@
+"""
+Brainstem mesh using a tapered cylinder
+"""
+
+from __future__ import division
+import copy
+from opencmiss.zinc.element import Element
+from opencmiss.zinc.node import Node
+from opencmiss.utils.zinc.field import Field, findOrCreateFieldCoordinates, findOrCreateFieldGroup, findOrCreateFieldNodeGroup, findOrCreateFieldStoredMeshLocation, findOrCreateFieldStoredString
+from opencmiss.utils.zinc.finiteelement import getMaximumNodeIdentifier
+from scaffoldmaker.annotation.annotationgroup import AnnotationGroup, findOrCreateAnnotationGroupForTerm
+from scaffoldmaker.annotation.brainstem_annotation_terms import get_brainstem_annotation_term
```

Comment:
You haven't added this new file; please also rename to brainstem_terms, get_brainstem_term, for consistency with other scaffolds.